### ISSUE TITLE
[BUG] Whitespaces in file names will fail the compilation

### ISSUE THREAD

**Issue by franneck94** [franneck94/Vscode-C-Cpp-Runner#76]

**Comment by franneck94** [comment]
I can't figure out how to get this working...
Somehow the vscode Task API does not like this

**Comment by franneck94** [comment]
Note: If the absolute path of the project dir that is opened in VSCode has whitespaces, this is find.
However, there the Compiler warnings are not mapped to the correct files in the problem tab.

See: #86

**Comment by GrossoLuca** [comment]
Hi, I think I could have solved the bug for Whitespaces giving error in compiling. 
At 
```
256  // Exe task
257  let objectFilesStr: string = '';
258  for (const objectfile of objectFiles) {
259    objectFilesStr += ` "${objectfile}"`;
260  }
```
I've tested with c++ projects on MacOS and it works

**Comment by franneck94** [comment]
Fixed in 5.2.2
